admin_manual/appliance/Clamav.rst

```diff
@@ -78,6 +78,11 @@ If you want to refresh the ClamAV database manually, follow these steps:
   # Start the refresh process again with this command:
   sudo freshclam
 
+..note::
+
+	- When the app is just enabled (without configuration or with wrong configuration) it will strictly reject **all** uploads for the whole instance.
```

Comment:
You can get lucky, if you are using "executable" mode and the ClamAV has been installed as an "executable", then the default settings "just work". That worked for me in testing yesterday. But it is true that in most situations the default settings will not "just work".

Reply:
The text is from @pmaier1 

I just copy/pasted

Reply:
That's fine - I was just pointing out that the statement is not 100% true. But anyway it is good that the admin knows that they have this issue, and if a few get lucky, then that is a bonus.

Reply:
sure